index.html

```diff
@@ -54,5 +55,148 @@ <h1>I’m Fadahunsi Glad to see you!</h1>
+              </p>
+              <ul class="work-technologies">
+                <li class="technology-tag">html</li>
+                <li class="technology-tag">css</li>
+                <li class="technology-tag">javascript</li>
+              </ul>
+              <button class="view-work-btn">See Project</button>
+            </div>
+            <div class="work-item">
+                <img alt="work-item-img" src="./img/work/work-2-cover.png"/>
+              <h3 class="work-title">Multi-Post Stories</h3>
+              <div class="work-subtitle">
+                <span>Canopy</span>
+                <span>Back End Dev</span>
+                <span>2015</span>
+              </div>
+              <p class="work-details">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <ul class="work-technologies">
+                <li class="technology-tag">html</li>
+                <li class="technology-tag">css</li>
+                <li class="technology-tag">javascript</li>
+              </ul>
+              <button class="view-work-btn">See Project</button>
+            </div>
+            <div class="work-item">
+              <img alt="work-item-img" src="./img/work/work-3-cover.png"/>
+              <h3 class="work-title">Tonic</h3>
+              <div class="work-subtitle">
+                <span>Canopy</span>
+                <span>Back End Dev</span>
+                <span>2015</span>
+              </div>
+              <p class="work-details">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <ul class="work-technologies">
+                <li class="technology-tag">html</li>
+                <li class="technology-tag">css</li>
+                <li class="technology-tag">javascript</li>
+              </ul>
+              <button class="view-work-btn">See Project</button>
+            </div>
+            <div class="work-item">
+                <img alt="work-item-img" src="./img/work/work-4-cover.png"/>
+              <h3 class="work-title">Multi-Post Stories</h3>
+              <div class="work-subtitle">
+                <span>Canopy</span>
+                <span>Back End Dev</span>
+                <span>2015</span>
+              </div>
+              <p class="work-details">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <ul class="work-technologies">
+                <li class="technology-tag">html</li>
+                <li class="technology-tag">css</li>
+                <li class="technology-tag">javascript</li>
+              </ul>
+              <button class="view-work-btn">See Project</button>
+            </div>
+          </section>
+          <section id="aboutMe" class="about-me">
+            <div class="top">
+              <h2 class="title">About Myself</h2>
```

Comment:
- [ ] Also here please try to use the right font for you title you should use 
![image](https://user-images.githubusercontent.com/61790587/170544599-5d147e9b-9840-4fac-8e71-c6252720946f.png)